<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>543</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>543</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>543</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="t_slider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>231</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>2200</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="changeparameter_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Parameter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Simulate_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Import_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Export_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current t [ms] = </string>
    </property>
   </widget>
   <widget class="QLabel" name="G">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="t">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="G_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current G = </string>
    </property>
   </widget>
   <widget class="QLabel" name="m_text">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>m = </string>
    </property>
   </widget>
   <widget class="QLabel" name="k_text">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>k = </string>
    </property>
   </widget>
   <widget class="QLabel" name="Tolerance_text">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tolerance = </string>
    </property>
   </widget>
   <widget class="QLabel" name="R_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>R = </string>
    </property>
   </widget>
   <widget class="QLabel" name="a_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>a = </string>
    </property>
   </widget>
   <widget class="QLabel" name="theta0_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>theta0 = </string>
    </property>
   </widget>
   <widget class="QLabel" name="psidot0_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>psidot0 = </string>
    </property>
   </widget>
   <widget class="QLabel" name="psidot0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="theta0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="R">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="a">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="m">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="k">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Tolerance">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Angles">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Angles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Derivatives">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Derivatives</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Positions">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Positions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Tecplot_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Tecplot</string>
    </property>
   </widget>
  </widget>
  <action name="actionOpenParameterOptions">
   <property name="text">
    <string>openParameterOptions</string>
   </property>
  </action>
  <action name="actionSlideTime">
   <property name="text">
    <string>SlideTime</string>
   </property>
  </action>
  <action name="actionAnglesClicked">
   <property name="text">
    <string>AnglesClicked</string>
   </property>
  </action>
  <action name="actionDerivativesClicked">
   <property name="text">
    <string>DerivativesClicked</string>
   </property>
  </action>
  <action name="actionPositionsClicked">
   <property name="text">
    <string>PositionsClicked</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>changeparameter_button</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenParameterOptions</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>t_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>actionSlideTime</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Angles</sender>
   <signal>clicked()</signal>
   <receiver>actionAnglesClicked</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Derivatives</sender>
   <signal>clicked()</signal>
   <receiver>actionDerivativesClicked</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Positions</sender>
   <signal>clicked()</signal>
   <receiver>actionPositionsClicked</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ExportbuttonClicked()</slot>
 </slots>
</ui>
